addons/mail/static/tests/owl/component/chatter_topbar_tests.js

```diff
@@ -110,6 +192,161 @@ QUnit.test('base rendering', async function (assert) {
     );
 });
 
+QUnit.test('base rendering (attachments loading transition)', async function (assert) {
+    assert.expect(13);
+
+    const attachmentPromise = makeTestPromise();
+    await this.start({
+        debug: true,
+        async mockRPC(route) {
+            if (route.includes('ir.attachment/search_read')) {
+                await attachmentPromise;
+                return [];
+            }
+            return this._super(...arguments);
+        }
+    });
+    const chatterLocalId = this.env.store.dispatch('createChatter', {
+        initialThreadId: 100,
+        initialThreadModel: 'res.partner',
+    });
+    await this.createChatterTopbar(chatterLocalId, { isDisabled: false });
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar`).length,
+        1,
+        "should have a chatter topbar"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonSendMessage`).length,
+        1,
+        "should have a send message button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonLogNote`).length,
+        1,
+        "should have a log note button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonScheduleActivity`).length,
+        1,
+        "should have a schedule activity button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonFollow`).length,
+        1,
+        "should have a follow button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonFollowers`).length,
+        1,
+        "should have a followers button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonFollowersCount`).length,
+        1,
+        "followers button should have a counter"
+    );
```

Comment:
Useless in this test.
I feel like test should rarely test everything. You may have one test with base rendering, but most tests should be very specific.